sklearn/ensemble/forest.py

```diff
@@ -242,6 +243,10 @@ def fit(self, X, y, sample_weight=None):
         -------
         self : object
         """
+        # Validate hyperparameters
+        if self.n_estimators == 10:
+            warnings.warn("'n_estimators' default value will be changed to 100 in version 0.22. For optimal performance, you should make change this value to 100", FutureWarning)
```

Comment:
I don't think that you should have the statement about optimal performance.

Also line length must be <80